src/SecTester.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -15,5 +19,28 @@ public static IServiceCollection AddSecTesterConfig(this IServiceCollection coll
       collection.Add(new ServiceDescriptor(typeof(Configuration), configuration));
       return collection;
     }
+
+    public static IServiceCollection AddSystemTimeProvider(this IServiceCollection collection, SystemTimeProvider? instance = null)
+    {
+      collection.AddSingleton(instance ?? new UtcSystemTimeProvider());
+      return collection;
+    }
+
+    public static IServiceCollection AddDefaultLogging(this IServiceCollection collection, LogLevel logLevel = LogLevel.Error)
+    {
+      
+      collection.AddLogging((builder) =>
+      {
+        builder.AddConsoleFormatter<ColoredConsoleFormatter, DefaultConsoleFormatterOptions>();
```

Comment:
For details see the comments above


```suggestion
        builder.AddConsoleFormatter<ColoredConsoleFormatter, ConsoleFormatterOptions>(options => {
          options.TimestampFormat = "yyyy-MM-dd HH:mm:ss.fff";
          // etc
        });
```